src/main/java/edu/hm/hafner/analysis/parser/DScannerParser.java

```diff
@@ -0,0 +1,100 @@
+package edu.hm.hafner.analysis.parser;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.util.Optional;
+import java.util.stream.StreamSupport;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+import edu.hm.hafner.analysis.Issue;
+import edu.hm.hafner.analysis.IssueBuilder;
+import edu.hm.hafner.analysis.ParsingException;
+import edu.hm.hafner.analysis.ReaderFactory;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+
+/**
+ * Parser report in JSON format as exported by DScanner.
+ *
+ * @author Andre Pany
+ */
+public class DScannerParser extends JsonBaseParser {
```

Comment:
Would it rather make sense to derive directly from `JsonParser`? Then we can reuse the methods `accepts` and `parse`?